libs/asset-viewer/src/lib/state/asset-search/asset-search.reducer.ts

```diff
@@ -78,6 +78,12 @@ export const assetSearchReducer = createReducer(
         ...state.query,
         ...query,
       },
+    })
+  ),
+  on(
+    actions.loadSearch,
```

Comment:
Why do we need this? Is this because of the loading state?
(Currently we have three different loading states, but actually just need one, because the loading bar is displayed as soon as one state is set to loading, right?) --> Then setting the filterLoadingState on actions.resetSearch can be removed, because we already set the loadingState on loadSearch